ArduPlane/Parameters.cpp

```diff
@@ -1543,6 +1542,15 @@ void Plane::load_parameters(void)
 #if AP_FENCE_ENABLED
     AP_Param::convert_class(g.k_param_fence, &fence, fence.var_info, 0, 0, true);
 #endif
+  
+    // PARAMETER_CONVERSION - Added: Dec 2023
+    // Convert _CM (centimeter) parameters to meters and _CD (centidegrees) parameters to meters
+    {
+        const AP_Param::ConversionInfo centimeters_conversion_info[] {
```

Comment:
we need to create a special method for this in AP_Param, so we can do:
  myparam.convert_centi_parameter();
and it is done

Reply:
Not really. That way you have to write a loop. This way you can pass in a whole table of parameters using the existing function - which is what I'm expecting next ...

```
        const AP_Param::ConversionInfo centimeters_conversion_info[] {
            { Parameters::k_param_pitch_trim_cd_old, 0, AP_PARAM_INT16, "PITCH_TRIM" },
            { Parameters::k_param_lim_pitch_max_cd_old, 0, AP_PARAM_INT16, "PITCH_LIM_MAX" },
            { Parameters::k_param_lim_pitch_min_cd_old, 0, AP_PARAM_INT16, "PITCH_LIM_MIN" },
            { Parameters::k_param_lim_roll_old, 0, AP_PARAM_INT16, "ROLL_LIM" },
            { Parameters::k_param_airspeed_cruise_old, 0, AP_PARAM_INT16, "AIRSPEED_CRUISE" },
        };
        AP_Param::convert_old_parameters_scaled(&centimeters_conversion_info[0], ARRAY_SIZE(centimeters_conversion_info), 0.01f, 0);

```

Reply:
> we need to create a special method for this in AP_Param, so we can do: myparam.convert_centi_parameter(); and it is done

I took a stab at it by copying the existing convert_parameter_width() method in AP_Param, it seems to work, but it's very low level so I wonder if you could take a look. I had to pass in the old type to get it to work.

Reply:
Tested in SITL